cinch/roles/jenkins_master/tasks/iptables.yml

```diff
@@ -10,6 +10,38 @@
     state: stopped
     enabled: false
 
+- name: set iptables to accept all icmp
```

Comment:
Can we add comments to these iptables rules so that if someone were inspecting a host configuration they could know the intent of these rules?